pkg/scheduler/core/generic_scheduler_test.go

```diff
@@ -1889,7 +1886,10 @@ func TestPickOneNodeForPreemption(t *testing.T) {
 			for _, f := range test.registerPlugins {
 				f(&registry, plugins, pluginConfigs)
 			}
-			fwk, _ := framework.NewFramework(registry, plugins, pluginConfigs, framework.WithSnapshotSharedLister(snapshot))
+			fwk, err := framework.NewFramework(registry, plugins, pluginConfigs, framework.WithSnapshotSharedLister(snapshot))
```

Comment:
Can you refactor it to be:

```suggestion
			fwk, err := st.NewFramework(test.registerPlugins, framework.WithSnapshotSharedLister(snapshot))
```
And remove L1879~L1888.

Reply:
Good catch, done